### ISSUE TITLE
Create Exercise report(s), similar to WorkoutsHistory.

### ISSUE THREAD

**Issue by jpdillingham** [jpdillingham/SWoT#73]
## Tabular View
* [x] Display a table with columns for each metric in the data set (map and reduce)
* [x] Map results to appropriate columns (columns as props, row factory? needs design)
* [x] Ability to filter by exercise by passing id in route

Exercise metrics may change over time.  Columns in the history table should be dynamic in that if an Exercise contains a metric, it should be displayed in a column.  Exercises in the same data set lacking that metric will have a blank column or potentially N/A.

**Comment by jpdillingham** [comment]
Blocked by #97

**Comment by jpdillingham** [comment]
Tabular view closed by #103

**Comment by jpdillingham** [comment]
Graph functionality split to a separate issue
